lib/column/json.go

```diff
@@ -150,7 +145,9 @@ func (col *Json) AppendRow(v interface{}) (err error) {
 				if col.columns[path], err = toType(reflect.TypeOf(add_v)).Column(); err != nil {
 					return err
 				}
-				col.columns[path].AppendRow(add_v)
+				if err := col.columns[path].AppendRow(add_v); err != nil {
```

Comment:
1. nit: err is the return parameter; could just assign as #145 instead of creating a new one with ":="

2. also check the AppendRow return error in #137 & #139